integration-tests/README.md

```diff
@@ -203,7 +117,25 @@ namespace: default
 
 Certain properties like weblogicDomainStoragePath, image, externalOperatorCert are populated at run time.
 
+# Input Yaml to the test
+
+The input yaml file(for example, domainonpvwlst.yaml) to the java test is used to override any or all the attributes in
+- samples domain inputs - click [here](https://github.com/oracle/weblogic-kubernetes-operator/blob/develop/kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-domain-inputs.yaml) to see samples create-domain-inputs file
+- PV/PVC inputs - click [here](https://github.com/oracle/weblogic-kubernetes-operator/blob/develop/kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/create-pv-pvc-inputs.yaml) to see PV/PVC inputs file
+- load balancer attributes and
+- custom attributes that are used to provide more flexibility for creating the domain.
+
+Below are the load balancer attributes:
+- loadBalancer: value can be TRAEFIK or VOYAGER, default is TRAEFIK
```

Comment:
please put values like "loadBalancer", "VOYAGER", etc., in back ticks

Reply:
Done